.docker/Dockerfile-py3.ckan

```diff
@@ -0,0 +1,47 @@
+FROM amazeeio/python:3.8-22.7.0
+
+ARG SITE_URL=http://ckan:3000/
+ARG CKAN_REPO=ckan/ckan
+ARG CKAN_VERSION=ckan-2.9.5
+ENV PYTHON_VERSION=py3
+ENV SITE_URL="${SITE_URL}"
+ENV VENV_DIR=/app/ckan/default
+ENV APP_DIR=/app
+ENV CKAN_INI=/app/ckan/default/production.ini
+
+WORKDIR "${APP_DIR}"
+
+ENV DOCKERIZE_VERSION v0.6.1
+RUN apk add --no-cache curl build-base git libxml2-dev libxslt-dev postgresql postgresql-client postgresql-dev \
```

Comment:
I'm guessing we could make this a common shell script between our two docker containers to just run, and need to have these commands 'inlined'?

Reply:
Containers are very similar across different projects, yes.